jwst/stpipe/tests/data/mkdata.py

```diff
@@ -1,18 +1,18 @@
-#!/usr/bin/env python
-import numpy
-from astropy.io import fits as pyfits
+import numpy as np
+from astropy.io import fits as fits
```

Comment:
Trivial nitpick, but why bother using an "as <name>" when <name> is the same as the package? Why not just leave it as "from astropy.io import fits"? The use in the subsequent code is identical.

Reply:
Doh!  I missed that.  Will change.

Reply:
Also taking Dan's suggestion and just wrapping the code in an `if` statement, so it can still be run as a script if needed.